### DISCUSSION TITLE
Backendmodule and CSV-Export

### DISCUSSION THREAD

**Discussion by micha68g** [oliverklee/ext-seminars#2589] [General]
Hi,
I was running an older version of seminars (3.4.x) , where I could export the registrations for an event from the BE-Module.

Now in Ver. 5.3.0 I don't have the 4 Tabs in the BE-Modul anymore (the manual says: "Back-end module “Events”: The back-end module "Events" has four different tabs: Events, Registrations, Speakers, Organizers"...so I think I missed some configuration-steps?!) 
There's just an event-list with some cols like "title", "date" "seats", "regstrations", "view" and "e-mail" but I don't have the CSV-Button (and the cancel-button) anymore in the row. (Manual says: "In the event list in the back-end module Events , you’ll find a button named CSV")
If I take the intermediate step via "view" and click "Download CSV" the file is empty....

What am I missing?

Thanks

Michael

**Comment by oliverklee** [comment]
If the folder selected in the page tree has any event records in it, the BE module has a CSV export button for the events in the top-left corner:

![grafik](https://github.com/oliverklee/ext-seminars/assets/765746/5e3e74a6-fc57-4419-8a8d-398e99344984)

For events that have at least one registration, you have a "show" button for the registrations in the table:

![grafik](https://github.com/oliverklee/ext-seminars/assets/765746/ba16b62d-1063-407b-ad72-84a7c30af1f7)

Then, in the list of registrations for this event, you have a button for exporting the listed registrations as CSV:

![grafik](https://github.com/oliverklee/ext-seminars/assets/765746/7f745dcf-0f16-4f66-9a32-be74cdb8170f)


Does this work for you?


The manual probably is out of date there (as I didn't update it after rewriting the BE module). Would you be willing to create a PR to update the manual accordingly?

**Comment by micha68g** [comment]
Thanks for the quick response. Yes, there are the "show" and "Download as CSV file" buttons) , but as I said: The downloaded file is always empty... 

Are these options still valid?
https://docs.typo3.org/p/oliverklee/seminars/5.3/en-us/Reference/SetupCommonForFront-endPlug-inAndBack-endModuleInPlugintxSeminars/Index.html#setup-common-for-front-end-plug-in-and-back-end-module-in-plugin-tx-seminars

fieldsFromEventsForCsv = 
and others

Thanks Michael

**Comment by oliverklee** [comment]
> Are these options still valid?

Yes, they are (or should be). When I last tried it, the CSV files (both for registrations and events) contained data.

Could you please check that these TypoScript settings are set when you go on the corresponding folder in the BE and check in the TypoScript object browser?

**Comment by micha68g** [comment]
Well, after checking again I now understand my mistake.
I have a 3 Domain/Root-Setup in one TYPO3-Installation all participating in the same "Events"-Folderstructure and I missed some configuration changes for the other 2 root-pages...

Thanks
Michael

**Reply by oliverklee** [reply]
Ah, that’s good to read that you got that solved!

Would you still be willing to fix the manual so the people after you won’t run into the same kind of confusion?
